advanced/workflows/ancient-egyptian-email.mdx

```diff
@@ -0,0 +1,176 @@
+---
+title: 'Building an Ancient Egyptian Email Assistant'
+description: 'Create an automated email response system with an Ancient Egyptian theme using Shinkai Tools'
+---
+
+# Building an Ancient Egyptian Email Assistant
+
+This tutorial guides you through creating an automated email response system using Shinkai Tools. We'll implement an assistant that responds to emails with the personality of an ancient Egyptian pharaoh, demonstrating the power and flexibility of Shinkai's tool system.
+
+## Prerequisites
+
+Before starting this tutorial:
+1. Install and open Shinkai Desktop
+2. Enable experimental features in Settings
+3. Have an email account with IMAP/SMTP access that supports secure connections (SSL/TLS)
+4. Enable two-factor authentication and generate an app-specific password if supported by your provider
+
+## Email Tool Configuration
+
+First, configure the Email Fetcher and Email Sender tools in the Shinkai Tools section. These are native tools that work out of the box - no additional imports required.
+
+### Email Fetcher (IMAP) Configuration
+Configure the following in the Tool Config page:
+- IMAP Server: Your email provider's IMAP server
+- Port: Your provider's IMAP port (typically uses SSL/TLS)
+- Username: Your email address
+- Password: Your email password or app-specific password
+
+### Email Sender (SMTP) Configuration
+Configure in the same Tool Config page:
+- SMTP Server: Your email provider's SMTP server
+- Port: Your provider's SMTP port (typically uses SSL/TLS)
+- Username: Your email address
+- Password: Your email password or app-specific password
+
+Note: Refer to your email provider's documentation for the specific server addresses and port numbers. Many providers also recommend using app-specific passwords for enhanced security.
+
+## Creating the Email Assistant
+
+Navigate to the Tool Editor through the Shinkai Tools section. Our implementation uses these native Shinkai tools:
+```typescript
```

Comment:
this snippet is out of place. PLease begin this section by explaining a little bit about the Shin kai Tools Editor. How to navigate to it by going to Sinkai Tools > Create new tool. Then explain that the editor needs an initial prompt in the left chat to enable code edition, then explain to the user it can paste the code in the code editor on the right.

The import snippets must be accompanied by explaining that the email answering tool is built using those four other tools